coffee-chats/src/main/webapp/src/components/ListItemLink.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import {Link, useRouteMatch} from "react-router-dom";
+import {ListItem, ListItemText} from "@material-ui/core";
+
+export function ListItemLink(props: {to: string, primary: string}) {
+  const {to, primary} = props;
```

Comment:
Move destructuring to parameters and create an interface for each component props